### PR TITLE
Double rendering

### PR DESCRIPTION
**Pull request by rubelux**

### CODE REVIEW

File: src/components/Layout/Header/AccountButton/index.tsx [hunk 1]
```diff
@@ -14,18 +14,32 @@ const AccountButton = ({ active, ...props }: SimpleComponentProps & { active: bo
   const { data: session, status } = useSession()
   const loggedInUser = useReactiveVar(loggedInUserVar)
```
Comment (geometrizen) [inline src/components/Layout/Header/AccountButton/index.tsx:15]: just checking that we still want `useReactiveVar` here?

File: src/lib/Intercom.ts [hunk 2]
```diff
@@ -18,11 +18,14 @@ export function initIntercom() {
 
   const appId = assert(process.env.NEXT_PUBLIC_INTERCOM_APP_ID, 'NEXT_PUBLIC_INTERCOM_APP_ID')
 
+  console.log('in here ', appId)
```
Comment (geometrizen) [inline src/lib/Intercom.ts:21]: do we still need this debug line?

**Review by geometrizen — APPROVED** [review]
LGTM worked locally

File: src/components/Layout/Header/AccountButton/index.tsx (hunk 1 above)
Comment (rubelux) [reply src/components/Layout/Header/AccountButton/index.tsx:15]: We'd need to do a lot of refactoring to stop using it

**Pull request merged**